internal/flink/command_artifact_create.go

```diff
@@ -58,10 +65,15 @@ func (c *command) createArtifact(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+	runtimeLanguage, err := cmd.Flags().GetString("runtime-language")
+	if err != nil {
+		return err
+	}
 
 	extension := strings.TrimPrefix(filepath.Ext(artifactFile), ".")
-	if strings.ToLower(extension) != "jar" {
-		return fmt.Errorf(`only ".jar" file extensions are allowed`)
+	ok := slices.Contains(allowedFileExtensions, extension)
+	if !ok {
+		return fmt.Errorf("only extensions allowed for --artifact-file are %v", utils.ArrayToCommaDelimitedString(allowedFileExtensions, "or"))
 	}
```

Comment:
I don't think we need this check since we have https://github.com/confluentinc/cli/pull/2780/files#diff-e513cee019b58f8060dcd5a7f48c9db3895a249d32f8498cabcccf9352d97a4aR53, right?